README.rst

```diff
@@ -10,3 +10,116 @@ flamingo
 
 Flamingo is a python3-based, `pelican <https://blog.getpelican.com/>`_-inspired
 static site generator.
+
+
+Basic Concepts
+--------------
+
+Flamingo aims to convert content written in a markup language into a static
```

Comment:
`a markup language` or `reStructuredText`? I assumed flamingo is intentionally rst-only?